### ISSUE TITLE
Unable to reproduce the output in brevitas using torch

### ISSUE THREAD

**Issue by YZW-explorer** [Xilinx/brevitas#345]
Dear author, hello! The model trained by brevitas works very well, this is a very good tool! In the past month, I was still trying to use the model built by torch.nn to load the parameters of the brevitas model to make the accuracy of the torch model and the brevitas model the same, but I encountered a problem and need your answer, thank you very much!
**This is part of my brevitas model. The data is different from this part:**
```
id = qnn.QuantIdentity(bit_width=4, return_quant_tensor=True),
qnnconv1 = qnn.QuantConv2d(inp, inp, kernel_size=3, stride=self.stride, padding=1, groups=inp,
                weight_bit_width=weight_bit_width, weight_quant_type=self.weight_qtype,bias=False,weight_quant=CommonIntWeightPerChannelQuant，return_quant_tensor=False)
```

**The following is the corresponding torch model:**
```
nnconv1 = nn.Conv2d(inp, inp, kernel_size=3, stride=self.stride, padding=1, groups=inp,
                bias=False)
```
I have made a simple representation of my implementation here:
nnconv1.weight = qnnconv1.int_weight()
Suppose the input is x
**torch model:**
```
def nnforward(self, x):
    x = x/qnnconv1.quant_input_scale()
    x = nnconv1(x)
    x = x*qnnconv1.quant_outputput_scale()
```
**brevitas model**
```
def qnnforward(self, x):
    x = id(x)
    x = qnnconv1 (x)
```

But nnforward(x) is not equal to qnnforward(x).why is that?
Looking forward to your reply!Thank you!

**Comment by YZW-explorer** [comment]
So it may be the problem of precision of data? The picture on the left prints the output of the pytorch model, and the picture on the right prints the output of the brevitas model. It can be seen that some very small numbers are regarded as 0 by pytorch, while brevitas keeps them.
How can I solve this problem?
![image](https://user-images.githubusercontent.com/74363764/124503443-24905a00-ddf8-11eb-9ef6-2dddb48507e6.png)
